{
    "id": "CVE-2021-28363",
    "product": "n/a",
    "version": "n/a",
    "vulnerability": [
        "n/a"
    ],
    "description": "The urllib3 library 1.26.x before 1.26.4 for Python omits SSL certificate validation in some cases involving HTTPS to HTTPS proxies. The initial connection to the HTTPS proxy (if an SSLContext isn't given via proxy_config) doesn't verify the hostname of the certificate. This means certificates for different servers that still validate properly with the default urllib3 SSLContext will be silently accepted.",
    "poc": {
        "reference": [
            "https://www.oracle.com/security-alerts/cpuoct2021.html"
        ],
        "github": [
            "https://github.com/noseka1/deep-dive-into-clair",
            "https://github.com/tern-tools/tern"
        ]
    }
}